images/scripts/opensuse-tumbleweed.setup

```diff
@@ -145,6 +145,18 @@ useradd -c Builder -G mock builder
 
 su builder -c "sudo mock --verbose -i $(/var/lib/testvm/build-deps.sh "${ID} ${VERSION_ID}")"
 
+# Setup redis
+sudo cp /etc/redis/default.conf.example /etc/redis/redis.conf
+sudo chown redis:redis /etc/redis/redis.conf
+
+mkdir -p /etc/systemd/system/redis.target.d
+cat > /etc/systemd/system/redis.target.d/70-redis-targets.conf <<EOF
+[Unit]
+Wants=redis@redis.service
+EOF
```

Comment:
Is that documented in TW somehow? That's all stuff that should either happen by default (to get an useful experience out of the box) or in Cockpit's metrics page. It doesn't help users if tests go green with "it works against a hacked up image", but not against a default install.